### PR TITLE
test_mtls_enterprise_without_client_cert stability fix.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline sugar pretty please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline sugar pretty please with a cherry on top

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline sugar pretty please with a sweet cherry on top

**Comment by kjaskiewiczz** [comment]
looks good

File: tests/tests/test_mtls.py [hunk 1]
```diff
@@ -356,8 +355,23 @@ def test_mtls_enterprise_without_client_cert(self, setup_ent_mtls):
...
-        time.sleep(30)
+        # in here it also may happen, that the client is started earlier, and device registers
+        # as pending. in that case get_devices_status which is called from get_devices will
+        # loop until it runs out of iterations, due to the fact that we expect to have 0 devices.
+        # to prevent this from happening, lets wait a bit, if the device shows as pending,
+        # if it does, decommission it, and then restart the client, and wait to be sure the
+        # device will not re-appear, which is the main idea of the test.
+        device_not_present_timeout_seconds = 30
+        for device in devauth.get_devices_status(
+            "pending", max_wait=device_not_present_timeout_seconds * 0.5, no_assert=True
```
Comment (tranchitella) [inline tests/tests/test_mtls.py:366]: Why do you set device_not_present_timeout_seconds to 30 (seconds?) and then multiply it by 0.5?

Comment (merlin-northern) [reply tests/tests/test_mtls.py:366]: > 

please find the reply below:)

**Review by kjaskiewiczz — APPROVED** [review]

**Comment by merlin-northern** [comment]
@tranchitella weekend is a good time to have it in, do you have any objections? have I answered your question to your satisfaction?

**Review by tranchitella — APPROVED** [review]

**Comment by merlin-northern** [comment]
thank you.

**Pull request merged**